ParquetClassLibrary/EntityID.cs

```diff
@@ -91,6 +92,27 @@ public bool IsValidForRange(Range<EntityID> in_range)
             return _id == None || in_range.ContainsValue(Math.Abs(_id));
         }
 
+        /// <summary>
+        /// Validates the current ID over a collection of <see cref="Utilities.Range{EntityID}"/>s.
+        /// An ID is valid if:
+        /// 1) it is <see cref="Sandbox.ID.EntityID.None"/>
+        /// 2) it is defined within the given range, regardless of sign.
+        /// </summary>
+        /// <returns><c>true</c>, if the identifier is valid given any of the given ranges, <c>false</c> otherwise.</returns>
+        /// <param name="in_ranges">The range within which the absolute value of the ID must fall.</param>
+        [Pure]
+        public bool IsValidForRange(IEnumerable<Range<EntityID>> in_ranges)
```

Comment:
Is the purpose of this to ask, "is this ID valid for all ranges in this set"? What's a use case for this scenario?

Reply:
So, I plan to use this in the forthcoming Item class to check if a purported parquet ID is valid for any of FloorIDs, BlockIDs, FurnishingIDs, or CollectibleIDs:
`if (!in_asParquet.IsValidForRange(AssemblyInfo.ParquetIDs))`

Reply:
Ah, so it's "is this ID valid for any range in this set", gotcha.